base_search_improved/README.rst

```diff
@@ -0,0 +1,93 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Improved Search
+===============
+
+This module extends the search functionality to add simple shortcuts for
+more complex search conditions.
+
+Features:
+* Fuzzy search: use wildcards ("%") in place of spaces
```

Comment:
You must use `*` instead of spaces for the wildcard, or even better, you are allowed to use also `%` in the searches, so better to teach users to use it, that to introduce a system that don't allow to make specific searches.